.github/workflows/.github-actions.yml

```diff
@@ -0,0 +1,25 @@
+name: pyRACF package
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r requirements-development.txt
```

Comment:
I think it is generally recommended to use `python3 -m pip` instead of just `pip`. It is also generally better to call `python3` instead of just `python`

Reply:
Changed to:

```
    - name: Install dependencies
      run: |
        python3 -m pip install --upgrade pip
        python3 -m pip install -r requirements.txt
        python3 -m pip install -r requirements-development.txt
```

Reply:
Looks good.